### ISSUE TITLE
Code generated following the example in the README is invalid

### ISSUE THREAD

**Issue by ernesto-jimenez** [goadesign/goa#231]
```
# goa-adder
./main.go:21: cannot use service (type "github.com/goadesign/goa".Service) as type "github.com/raphael/goa".Service in argument to NewOperandsController:
        "github.com/goadesign/goa".Service does not implement "github.com/raphael/goa".Service (wrong type for ErrorHandler method)
                have ErrorHandler() "github.com/goadesign/goa".ErrorHandler
                want ErrorHandler() "github.com/raphael/goa".ErrorHandler
./operands.go:16: cannot use OperandsController literal (type *OperandsController) as type app.OperandsController in return argument:
        *OperandsController does not implement app.OperandsController (missing Crit method)
```

**Comment by raphael** [comment]
Is it possible that you would have an old version of `goa` somewhere? Looks like the generated generator is using the old package path which means it must have been compiled with the old code.

**Comment by sheepbao** [comment]
I got this:
../../../../work/listomego/src/github.com/goadesign/goa/design/dsl/action.go:6:2: cannot find package "bitbucket.org/pkg/inflect" in any of:
	/home/lstbao/software/go/src/bitbucket.org/pkg/inflect (from $GOROOT)
	/home/lstbao/work/listomego/src/bitbucket.org/pkg/inflect (from $GOPATH)
	/home/lstbao/gowork/src/bitbucket.org/pkg/inflect

**Comment by raphael** [comment]
@sheepbao please run `make depend` that should get all the required packages.

**Comment by raphael** [comment]
@ernesto-jimenez thanks a lot for reporting then fixing the issue! Fixed by #234
